### DISCUSSION TITLE
starship.toml not getting parsed

### DISCUSSION THREAD

**Discussion by bdemick** [starship/starship#1503] [Q&A] [answered]
I just installed Starship in a new Ubuntu 20.04 VM, and made some initial changes to `starship.toml`, but I'm not seeing them reflected in the prompt at all: 

![Screen Shot 2020-07-22 at 2 17 00 PM](https://user-images.githubusercontent.com/3675695/88213397-b4a8b400-cc26-11ea-8a25-56a79e006a45.png)

`.bashrc` is properly configured to initialize Starship, and you can see from the image above that the Starship default prompt is being used, indicating that Starship is initialized properly. However, it doesn't look like the add_newline or character settings are being applied. What am I missing? On macOS the prompt gets updated as soon as the starship.toml file is saved.

**Comment by andytom** [comment]
Hi @bdemick, at a guess this is going to be something wrong with your config file, can you run the following it should help to pinpoint what the issue is.
```bash
STARSHIP_LOG=debug starship prompt
```

**Comment by matchai** [comment, marked as answer]
At a glance, it looks like it may be caused by the angular quotes in your config file: `“”` vs `""`.

**Reply by bdemick** [reply]
Thanks all! The angular quotes were the problem (probably from copying out of Slack), and that was also indicated by the debug logs. Much appreciated!
